drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -6,6 +6,8 @@
 
 # frozen_string_literal: true
 
+# THIS FILE IS GENERATED, DO NOT EDIT DIRECTLY
```

Comment:
It's true this is generated, but I think I intentionally left this comment out because we did have the idea that we may need to edit this manually after generation. (Generation should only happen when data standards change).

For example, I would suggest making a manual change to revert the `HUD_ESG_CV_DEPRECATED ` enum back to `HUD_ESG_CV`, so we don't introduce a breaking change in the API. (It's not a huge deal, but this is a breaking change because the frontend would continue to submit the old enum value until the new version is deployed).
I would suggest making that change with all enum changes where `_DEPRECATED` was added to the actual enum.